Cargo.toml

```diff
@@ -48,6 +48,11 @@ serde_json = "1.0.78"
 test_infra = { path = "test_infra" }
 wait-timeout = "0.2.0"
 
+[[test]]
```

Comment:
Yes, the `test` target defined in this way will be covered by `cargo fmt/clippy` ( which is how I make sure the format is correct during development :).

The double bracket is a toml syntax that represents "array", so that you can define multiple custom `test` targets in your toml file [1][2].

[1] https://doc.rust-lang.org/cargo/reference/manifest.html
[2] https://doc.rust-lang.org/cargo/reference/cargo-targets.html#configuring-a-target